src/main/kotlin/de/bund/digitalservice/useid/credentials/CredentialController.kt

```diff
@@ -0,0 +1,87 @@
+package de.bund.digitalservice.useid.credentials
+
+import de.bund.digitalservice.useid.events.AuthenticateEvent
+import de.bund.digitalservice.useid.events.EventService
+import de.bund.digitalservice.useid.events.EventType
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RestController
+import java.util.UUID
+
+internal const val CREDENTIALS_BASE_PATH = "/api/v1/credentials"
+
+@RestController
+@ConditionalOnProperty(name = ["features.desktop-solution-enabled"], havingValue = "true")
+class CredentialController(
+    private val credentialService: CredentialService,
+    private val eventService: EventService,
+) {
+    @PostMapping(path = [CREDENTIALS_BASE_PATH])
```

Comment:
Let's add correct swagger docs right away:
```
     @ResponseStatus(HttpStatus.ACCEPTED)
     @Operation(summary = "Start registration for webauthn credentials")
     @ApiResponse(responseCode = "202", content = [Content()])
```

Reply:
Thanks, will do. Returning 202 makes no sense here, right? The entity is created right away, so we should return 201.

Reply:
Done